src/openforms/registrations/contrib/objects_api/templates/registrations/contrib/objects_api/content_json.txt

```diff
@@ -0,0 +1,16 @@
+{% load registrations.contrib.objects_api.json_summary capture_tags %}
+{% capture as tt_openblock silent %}{% templatetag openblock %}{% endcapture %}
+{% capture as tt_closeblock silent %}{% templatetag closeblock %}{% endcapture %}
+{% capture as tt_openvariable silent %}{% templatetag openvariable %}{% endcapture %}
+{% capture as tt_closevariable silent %}{% templatetag closevariable %}{% endcapture %}
+{% autoescape off %}
+{
+  "data": {{ tt_openblock }} json_summary {{ tt_closeblock }},
+  "type": "terugbelnotitie",
```

Comment:
this should definitely be configurable too if we support templating here now

1. Add django-solo model default
2. Make it an optional field in the options serializer too